File: Estrutura de Decisao/Exercicio 4/Saudacao.php
<?php

/*
Verificar a hora do dia e dar a saudação correta:
Bom dia;
Boa tarde;
Boa noite
*/


$hora_especifica = time("G:i");
$hora = time("G");
$hora  = intval($hora);

verificar($hora, $hora_especifica);

function verificar($hora, $hora_especifica)
{
    if (6 < $hora && $hora <= 12) {
        echo "São " . $hora_especifica . " Bom dia.";
    } else if (13 <= $hora && $hora < 19) {
        echo "São " . $hora_especifica . " Boa tarde.";
    } else if (19 < $hora && $hora < 6) {
        echo "São " . $hora_especifica . " Boa noite.";
    }
}

File: Estrutura de Decisao/Exercicio 4/Saudacao.py
'''
Verificar a hora do dia e dar a saudação correta:
Bom dia;
Boa tarde;
Boa noite
'''
import datetime as dt


def verifica(hora: int, hora_especifica: str):
    if 6 < hora < 13:
        print("São {}. Bom dia.".format(hora_especifica))
    elif 13 < hora < 19:
        print("São {}. Boa tarde.".format(hora_especifica))
    elif 19 < hora < 6:
        print("São {}. Boa noite.".format(hora_especifica))


hora_especifica = dt.datetime.now()
hora_especifica = hora_especifica.strftime('%H:%M')
hora = dt.datetime.now().hour

verifica(hora, hora_especifica)
